src/test/codegen/sanitizer-cfi-emit-type-metadata-id-itanium-cxx-abi.rs

```diff
@@ -425,6 +429,8 @@ pub fn foo145(_: Type13, _: Type13) { }
 // CHECK: define{{.*}}foo145{{.*}}!type ![[TYPE145:[0-9]+]]
 pub fn foo146(_: Type13, _: Type13, _: Type13) { }
 // CHECK: define{{.*}}foo146{{.*}}!type ![[TYPE146:[0-9]+]]
+pub fn foo147(_: Type14<u8>) { }
+// CHECK: define{{.*}}foo147{{.*}}!type ![[TYPE30]]
```

Comment:
Would you also mind also using an user-defined concrete type for this similarly to Type12 so you have independent type ids for these functions?

Reply:
Done.